src/logservice/logminer/ob_log_miner_record_converter.cpp

```diff
@@ -30,13 +31,15 @@ namespace oblogminer
 {
 
 #define MINER_SCHEMA_DEF(field, id, args...) \
-ObLogMinerRecordCsvConverter::ColType::field,
-const ObLogMinerRecordCsvConverter::ColType ObLogMinerRecordCsvConverter::COL_ORDER[] =
+ILogMinerRecordConverter::ColType::field,
+const ILogMinerRecordConverter::ColType ILogMinerRecordConverter::COL_ORDER[] =
 {
   #include "ob_log_miner_analyze_schema.h"
 };
 #undef MINER_SCHEMA_DEF
 
+const char *ILogMinerRecordConverter::DELIMITER = ",";
```

Comment:
I don't think it's particularly appropriate to put DELIMITER here, as other output formats may not use it. But it's fine to put it here currently.